cypress/integration/like.spec.ts

```diff
@@ -2,23 +2,20 @@ it('likes an episode then finds it in the library', () => {
   cy.visitAndWaitForHydration('/podcasts/1');
 
   // Find a card and give it a name
-  cy.get('[data-cy="episode-card"]').first().as('episodeCard');
+  cy.findAllByTestId('episode-card').first().as('episodeCard');
```

Comment:
That makes sense. But also instead of liking all of them, maybe we should like random episodes and check for them?

Reply:
I don't see much of a reason to differentiate, it happens automatically anyway. I understand that it increases the runtime of our tests, but oh well

Reply:
I just thought it would be more human-like, because almost no one would like every single episode. If we also check that there are no episodes there that we didn't like (as we probably should), this will protect against possible bugs such as simply displaying all episodes there are.